scripts/generate-f32-igemm.sh

```diff
@@ -367,6 +367,14 @@ tools/xngen src/f32-igemm/avx512-broadcast.c.in -D MR=6 -D NR=16 -o src/f32-igem
 tools/xngen src/f32-igemm/avx512-broadcast.c.in -D MR=7 -D NR=16 -o src/f32-igemm/gen/f32-igemm-7x16-minmax-avx512f-broadcast.c &
 tools/xngen src/f32-igemm/avx512-broadcast.c.in -D MR=8 -D NR=16 -o src/f32-igemm/gen/f32-igemm-8x16-minmax-avx512f-broadcast.c &
 
+################################ RISC-V Vector ################################
+tools/xngen src/f32-igemm/MRxNRv-rvv.c.in -D MR=1 -D NR=m4 -D ACTIVATION=LINEAR -o src/f32-igemm/gen/f32-igemm-1x4v-rvv.c &
+tools/xngen src/f32-igemm/MRxNRv-rvv.c.in -D MR=7 -D NR=m4 -D ACTIVATION=LINEAR -o src/f32-igemm/gen/f32-igemm-7x4v-rvv.c &
+tools/xngen src/f32-igemm/MRxNRv-rvv.c.in -D MR=1 -D NR=m4 -D ACTIVATION=RELU   -o src/f32-igemm/gen/f32-igemm-1x4v-relu-rvv.c &
+tools/xngen src/f32-igemm/MRxNRv-rvv.c.in -D MR=7 -D NR=m4 -D ACTIVATION=RELU   -o src/f32-igemm/gen/f32-igemm-7x4v-relu-rvv.c &
+tools/xngen src/f32-igemm/MRxNRv-rvv.c.in -D MR=1 -D NR=m4 -D ACTIVATION=MINMAX -o src/f32-igemm/gen/f32-igemm-1x4v-minmax-rvv.c &
+tools/xngen src/f32-igemm/MRxNRv-rvv.c.in -D MR=7 -D NR=m4 -D ACTIVATION=MINMAX -o src/f32-igemm/gen/f32-igemm-7x4v-minmax-rvv.c &
```

Comment:
Normally we'd do a few more sizes so benchmarks on different ML models or cpus can be compared.
These are likely the right choice, in my experience, so its fine if youre comfortable with 7x4.
In my experience 7 is best on machines with an 8 way cache.  7 for the inputs and 1 for the weights.
4v is best on cpus with 4 vector units, but it fine for 2 vector units.
In the past we use MR=6 for cpus with 2 or 3 vector units, but I'm finding short MR and wide NR scales better with more vector units, larger memory and more complex gemms that support 4 bit.
This gemm/igemm, for example, has the option to support 4 or 8 bit weights on arm and intel.  Each weight vector gets converted...e.g.  int8 to float.  That works best with a wide but short kernel.
The relu and linear variations have only been used with wasmsimd in the past, due to very expensive min/max instructions.  You dont really need to implement them - minmax will usually be a very small impact on overall performance.
It would be nice in the future if cpu included a since vminmax instruction or a clamp or convert that takes parameters.  The qs8 gemm internally accumulates int, converts to float, scales, clamps to min/max values and converts to int8.  Its outside the loop but steps could be saved.